index.html

```diff
@@ -0,0 +1,232 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Crowdstaffing signup page</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="icon" href="photo/icon.png" type="image/icon type">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
```

Comment:
@indalgit in previous task I have asked to download the files and stored in local js file ? why you again didn't do that ?